package.json

```diff
@@ -47,7 +48,8 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "gulp": "^4.0.2",
-    "pre-commit": "^1.2.2"
+    "pre-commit": "^1.2.2",
+    "rimraf": "^3.0.0"
```

Comment:
rimraf is a part of appium-support module, which is already listed as dependency